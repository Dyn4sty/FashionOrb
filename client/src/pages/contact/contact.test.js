import React from "react";
import { shallow } from "enzyme";
import ContactPage from "./contact";

it("should render ContactPage component", () => {
  expect(shallow(<ContactPage />)).toMatchSnapshot();
});
