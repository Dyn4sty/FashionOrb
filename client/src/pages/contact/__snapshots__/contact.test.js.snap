// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ContactPage component 1`] = `
<styled.div>
  <h5>
    GOT A QUESTION?
  </h5>
  <h1
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    Contact FashionOrb
  </h1>
  <p>
    Weâ€™re here to help and answer any question you might have. We look forward to hearing from you
     
    <span
      aria-label="emoji"
      role="img"
    >
      ðŸ™‚
    </span>
  </p>
  <styled.div>
    <div
      className="card"
    >
      <p>
        Email:
         
        <b>
          <a
            href="mailto:example@gmail.com"
          >
            example@gmail.com
          </a>
        </b>
      </p>
      <p>
        Twitter:
         
        <b>
          <a
            href="#home"
          >
            Twitter Profile
          </a>
        </b>
      </p>
      <p>
        LinkedIn:
         
        <b>
          <a
            href="#home"
          >
            Linked Profile
          </a>
        </b>
      </p>
    </div>
  </styled.div>
  <iframe
    allowfullscreen=""
    aria-hidden="false"
    frameborder="0"
    height="600"
    src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3648.449330021458!2d90.3993155844375!3d23.873680384528818!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3755c59f42d61cf7%3A0xc440f706d8c5513e!2sFashion%20ORB!5e0!3m2!1siw!2sil!4v1586175686396!5m2!1siw!2sil"
    style={
      Object {
        "border": 0,
        "width": "100%",
      }
    }
    tabindex="0"
    title="GoogleMap"
    width="450"
  />
</styled.div>
`;
